src/main/java/org/apache/commons/cli/help/TextStyle.java

```diff
@@ -0,0 +1,341 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package org.apache.commons.cli.help;
+
+import java.util.function.Supplier;
+
+import org.apache.commons.cli.Util;
+
+/**
+ * The definition for styling blocks of text.  Most common usage is to style columns in a table, but may also be used to
+ * specify default stylings for a {@link HelpWriter}.
+ */
+public final class TextStyle {
+    /** The unset value for maxWidth */
+    public static final int UNSET_MAX_WIDTH = Integer.MAX_VALUE;
+    /** The alignment possibilities */
+    public enum Alignment {
+        /** left justify the text */
+        LEFT,
+        /** center the text */
+        CENTER,
+        /** right justify the text */
+        RIGHT }
+    /** Types scaling */
+    public enum Scaling {
+        /** do not scale */
```

Comment:
The Javadoc should say what the thing does, not what it doesn't do. Especially since the value is called "FIXED" and not "UNSCALED".